lib/buildmodel.js

```diff
@@ -135,6 +136,61 @@ const BuildModel = class {
     stream(config, response) {
         return this.executor.stream(config, response);
     }
+
+    /**
+     * Get a build based on id
+     * @param  {String}   id                The key of the record to retrieve
+     * @return {Function} callback
+     */
+    get(id, callback) {
+        const config = {
+            table,
+            params: {
+                id
+            }
+        };
+
+        return this.datastore.get(config, callback);
+    }
+
+    /**
+     * List builds with pagination
+     * @param  {Object}   paginate                  Config object
+     * @param  {Number}   paginate.count     Number of items per page
+     * @param  {Number}   paginate.page      Specific page of the set to return
+     * @return {Function} callback
+     */
+    list(paginate, callback) {
+        const config = {
+            table,
+            params: {},
+            paginate: {
+                count: paginate.count,
+                page: paginate.page
+            }
+        };
+
+        return this.datastore.scan(config, callback);
+    }
+
+    /**
+     * Update a build
+     * @param  {Object}    config         Config object
+     * @param  {String}    config.id      The key of the record to retrieve
+     * @param  {Object}    config.data    The new data object to update with
+     * @return {Function}  callback
+     */
+    update(config, callback) {
```

Comment:
Who's responsibility should it be to make sure that the build exists before updating?